waku/v2/rendezvous/rendezvous.go

```diff
@@ -162,23 +130,38 @@ func (r *Rendezvous) callRegister(ctx context.Context, rendezvousClient rvs.Rend
 	return t, retries
 }
 
-func (r *Rendezvous) Register(ctx context.Context, topic string) {
-	for _, m := range r.rendezvousPoints {
+func (r *Rendezvous) Register(ctx context.Context, rendezvousPoints []*RendezvousPoint) {
+	r.RegisterWithTopic(ctx, protocol.DefaultPubsubTopic().String(), rendezvousPoints)
+}
+
+func (r *Rendezvous) RegisterShard(ctx context.Context, cluster uint16, shard uint16, rendezvousPoints []*RendezvousPoint) {
```

Comment:
Improvement: Wondering if we should create a type called shard which contains cluster and shard, and abstracted out from all function calls. 
Later when we move to auto-sharding, then we don't need to change all functions..but just modify the type and add a function to it to compute the pubSubTopic.

Reply:
`RegisterRelayShards(ctx context.Context, rs protocol.RelayShards, rendezvousPoints []*RendezvousPoint)` receives this kind of structure you suggest. Perhaps later we can remove this `RegisterShard` function and leave the function that receives a `RelayShard`. I'm not sure if we should remove now as we haven't really used these functions and might end up deciding that passing the specific cluster and shard ends up being a better dev experience :thinking: